CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# main codeowner @paritytech/tools-team
+* @paritytech/tools-team
+
+# CI
+/.github/ @paritytech/ci @paritytech/tools-team
```

Comment:
you need to add @paritytech/ci as maintainer to this repo

Reply:
Done :) Thanks!